src/main/java/net/sourceforge/tess4j/Tesseract.java

```diff
@@ -199,22 +212,38 @@ public String doOCR(File imageFile) throws TesseractException {
     /**
      * Performs OCR operation.
      *
-     * @param imageFile an image file
+     * @param inputFile an image file
      * @param rect the bounding rectangle defines the region of the image to be
      * recognized. A rectangle of zero dimension or <code>null</code> indicates
      * the whole image.
      * @return the recognized text
      * @throws TesseractException
      */
```

Comment:
@nguyenq the changes of this file are safe and makes sende for me, though he is implementing the logic which every tess4j api consumer should do on his own in order to maintain flexibility, it's your call to decide.